vagrant_repo/cookbooks/cerner_splunk_test/recipes/_configure_guids.rb

```diff
@@ -0,0 +1,28 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk_test
+# Recipe:: _configure_guids
```

Comment:
It's meant to be on the run_list directly not called by the cerner_splunk_test cookbook... this shouldn't be an underscore recipe. (admittedly just for testing the cerner_splunk cookbook, but that's the convention.